wpf/README.md

```diff
@@ -16,7 +16,7 @@ Coming soon!
 
 ### Prerequisites and getting the tools
 
-Install Visual Studio 2017 Update 15.8 or higher from [https://visualstudio.microsoft.com/downloads/](https://visualstudio.microsoft.com/downloads/), selecting the **.NET desktop development** workload with the options: **.NET Framwork 4.7.2 development tools** and **.NET Core 2.1 development tools**. 
+Install Visual Studio 2017 Update 15.8 or higher from [https://visualstudio.microsoft.com/downloads/](https://visualstudio.microsoft.com/downloads/), selecting the **.NET desktop development** workload with the options: **.NET Framwork 4.7.2 development tools** and **.NET Core 2.1 development tools**.
```

Comment:
It is not possible to install the .NET Core 3.0 development tools via Visual Studio, so that would be quite misleading. However the very next sentence mentions how to install the 3.0 SDK.